src/components/molecules/EventPaymentButton/EventPaymentButton.scss

```diff
@@ -1,14 +0,0 @@
-@import "scss/constants.scss";
-
-.event-payment-button-container {
```

Comment:
Have we searched to ensure that this class (or any of the others in this file) is no longer referenced anywhere in the remaining app code? If so, please say so, then feel free to mark this comment as resolved.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40